index2.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style2.css" />
+
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="header">
+      <img class="cell-info" src="Light.png" alt="cell phone info" />
+    </div>
+    <div class="bottom-section">
+      <p class="welcome">Welcome</p>
+      <img class="union" src="Union.png" alt="union photo" />
```

Comment:
- [ ] Please use button tag for hamburger icon since you will add functionality to it in the upcoming projects